Cakefile

```diff
@@ -0,0 +1,81 @@
+# https://github.com/jcampbell05/xcake
+# http://www.rubydoc.info/github/jcampbell05/xcake/master/file/docs/Cakefile.md
```

Comment:
Did you use xcake to generate the project or just copy/modify the remote-config repo?

Reply:
I used xcake. Although I modified schemes to react to "Cmd+U" command.